### ISSUE TITLE
Track documentation views as metric

### ISSUE THREAD

**Issue by jeromekelleher** [tskit-dev/tskit#960]
In this [blog post](https://blog.dask.org/2020/01/14/estimating-users) Matthew Rocklin talks about how they estimate the number of users Dask has. The number of unique IPs looking at the documentation seems like quite a good metric to me.

Can we do something similar for tskit-dev related projects? (Since we're moving away from RTD?)

**Comment by benjeffery** [comment]
Yep, very doable - although we may need a cookie consent banner depending on how we do it. I'd like to try and avoid just dropping in google analytics javascript as it is pretty heavyweight tracking, and blocked by many developer-types.

**Comment by castedo** [comment]
> may need a cookie consent banner depending on how we do it

@benjeffery I use google analytics for Gene Heritage via the usual javascript. After turning off various types of data collection decided I was ok not having a cookie consent banner. I'm not a lawyer but felt comfortable enough with my understanding of GDPR etc... to believe a cookie consent banner was not needed given the google settings I opted for. The data I get is plenty informative and I don't miss opting out of PII. Feel free to ping me if you have any questions or want to compare/review settings, etc...
